build-yaml/econet-etwh-esp32.yaml

```diff
@@ -0,0 +1,13 @@
+---
+substitutions:
+  tx_pin: GPIO19
+  rx_pin: GPIO22
+  platform: esp32
+  board: esp32dev
+
+packages:
+  econet-etwh: !include ../econet_electric_tank_water_heater.yaml
+
+dashboard_import:
+  package_import_url: github://esphome-econet/esphome-econet/build-yaml/${name}-${platform}.yaml@${github_ref}
```

Comment:
Remove the dashboard_import section from ../econet_electric_tank_water_heater.yaml and all other files